Source/NSTableView.m

```diff
@@ -3216,18 +3234,23 @@ - (NSTableViewGridLineStyle) gridStyleMask
 - (NSCell *) preparedCellAtColumn: (NSInteger)columnIndex row: (NSInteger)rowIndex
 {
   NSCell *cell = nil;
-  NSTableColumn *tb = [_tableColumns objectAtIndex: columnIndex];
 
-  if ([_delegate respondsToSelector: 
-        @selector(tableView:dataCellForTableColumn:row:)])
-    {
-      cell = [_delegate tableView: self dataCellForTableColumn: tb
-                                                           row: rowIndex];
-    }
-  if (cell == nil)
-    {
-      cell = [tb dataCellForRow: rowIndex];
+  if (_viewBased == NO)
```

Comment:
Most likely this is the condition you wanted to use for NSOutlineView as well.

Reply:
These methods are identical except for the call to the table/outline view methods for the non-view based method.

Reply:
The code in NSOutlineView looks like this `if (_viewBased)`. Hopefully you see now that the part `== NO` is missing there. If you still don't see it, I am giving up here. I tried to explain it four times now. I cannot thing of a fifth way to do it.

Reply:
Ahhhh!!! *FACEPALM*. I see I thought, for some reason, you were referring to a different section of code.  Fixed.